project.clj

```diff
@@ -13,14 +13,17 @@
                  [org.postgresql/postgresql "9.4.1208.jre7"]
                  [com.zaxxer/HikariCP "2.4.3"]
                  [puppetlabs/kitchensink ~ks-version]
+                 [puppetlabs/i18n "0.4.0"]
                  [io.dropwizard.metrics/metrics-core "3.1.2"]
                  [io.dropwizard.metrics/metrics-healthchecks "3.1.2"]]
 
   :profiles {:dev {:dependencies [[org.slf4j/slf4j-api "1.7.21"]
                                   [org.slf4j/slf4j-log4j12 "1.7.21"]
                                   [log4j/log4j "1.2.17"]]}}
 
-  :plugins [[lein-release "1.0.5"]]
+  :plugins [[org.clojure/clojure "1.8.0"]
```

Comment:
This is needed to fix a version conflict between lein-release and i18n.  Note that this same version is applied to the whole project so it seemed pretty safe.